### PR TITLE
EVA-1898 Implement VcfReader that includes ID column (2/2)

### PR DESCRIPTION
**Pull request by jmmut**
this requires another PR in variation commons: https://github.com/EBIvariation/variation-commons/pull/91

### CODE REVIEW

File: eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/batch/tasklets/reportCheck/CoordinatesVcfFactory.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
         return variants;
     }
 
-    private Set<String> getIds(String[] fields) {
+    protected Set<String> getIds(String[] fields) {
```
Comment (andresfsilva) [inline eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/batch/tasklets/reportCheck/CoordinatesVcfFactory.java]: I think this is only being user in this class, why change it to protected?

File: eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/batch/processors/VariantToSubmittedVariantEntityProcessorTest.java [hunk 2]
```diff
@@ -45,15 +50,26 @@ public void setUp() {
...
+        // when
+        SubmittedVariantEntity processedVariant = processor.process(variant);
+
+        // then
         assertEquals(expectedSubmittedVariantEntity, processedVariant);
         assertEquals(expectedSubmittedVariantEntity.getAccession(), processedVariant.getAccession());
     }
+
+    @Test
+    public void ssidMissing() {
```
Comment (andresfsilva) [inline eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/batch/processors/VariantToSubmittedVariantEntityProcessorTest.java]:
```suggestion
    public void ssIdMissing() {
```

Comment (andresfsilva) [inline eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/batch/processors/VariantToSubmittedVariantEntityProcessorTest.java]:
```suggestion
    public void ssIdMissing() {
```

File: eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/batch/tasklets/reportCheck/CoordinatesVcfFactory.java (hunk 1 above)
Comment (jmmut) [reply eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/batch/tasklets/reportCheck/CoordinatesVcfFactory.java]: compilation failed as I added this method to the parent as protected (like the other methods for parsing each column. I should probably mark this as `@Override`.

**Comment by andresfsilva** [comment]
Suggest to rebase before merging given that #200 was already merged

**Pull request merged**